ou_dedetai/tui_curses.py

```diff
@@ -2,60 +2,74 @@
 import signal
 import textwrap
 
-from . import config
-from . import msg
-from . import utils
+from ou_dedetai import tui_screen
+from ou_dedetai.app import App
 
 
-def wrap_text(app, text):
+# NOTE to reviewer: does this convay the original meaning?
+# The usages of the function seemed to have expected a list besides text_centered below
+# Which handled the string case. Is it faithful to remove the string case?
```

Comment:
The main purpose of this function is to break up a line that is too long, otherwise it will cause curses to throw an error for trying to draw off screen. Eventually I implemented a catch-all try/except for these errors, but I attempted to fix them before doing so as much as possible.

Reply:
sounds good, removing comment